lib/sycamore/sycamore/tests/integration/connectors/opensearch/test_pdf_to_opensearch.py

```diff
@@ -97,7 +98,7 @@ def test_pdf_to_opensearch_with_llm_caching():
 
         """
 
-    s3_cache_base_path = os.environ["SYCAMORE_S3_TEMP_PATH"]
+    s3_cache_base_path = os.environ.get("SYCAMORE_S3_TEMP_PATH", "s3://aryn-sycamore-integ-temp/")
```

Comment:
It definitely feels like the S3 cache path should be part of the config object

Reply:
Yeah will migrate these in follow up PRs feature wise